addons/refuel/CfgVehicles.hpp

```diff
@@ -1,3 +1,5 @@
+#define XEH_INHERITED class EventHandlers {class CBA_Extended_EventHandlers: CBA_Extended_EventHandlers {};}
```

Comment:
Is this really needed?

Reply:
macro or its using?

Reply:
Both.

Reply:
Macro: 1 line instead of 4. `(4-1)*7 - 1 = ` -20 lines from file. I think it's worth it. I would really prefer this macro in CBA like XEH_ENABLED.
Using: it handles pumps created during mission with `createVehicle` after mission init.